paying_for_college/templates/worksheet.html

```diff
@@ -1410,7 +1689,7 @@ <h4 class="aid-form_summary-heading">
                                             $
                                         </span>
                                         <span class="line-item_amount"
-                                        data-financial="totalRepayment"
+                                        data-financial="totalDebt"
```

Comment:
Why this change in attribute name? Especially considering that in the copy of the page, "total debt" refers to the debt from a single offer for one year. The summary box right above this uses the exact phrasing to refer to a totally different number.

It might get confusing.